funnel/views/index.py

```diff
@@ -128,19 +128,17 @@ def home(self) -> ReturnRenderWith:
 IndexView.init_app(app)
 
 
-@app.route('/past.json')
+@app.route('/past')
```

Comment:
This will block 'past' as a username, so it either has to be in the reserved list in `models/helpers.py`, or the URL must have a character that is not valid in usernames (like a `.`)

Reply:
Changed it to past_projects